django-server/README.md

```diff
@@ -0,0 +1,16 @@
+# Installation instructions:
+
+### "The following apt commands will get you the packages you need:"
+#### sudo apt-get update
+#### sudo apt-get install python3-pip 
+
+#### and either 
+sudo apt-get install python3-venv
+#### or
+python3 -m pip install --user virtualenv
+
+## now you should be able to run the server-env-setup.sh script successfully.
+
+## For django + postgres installation check instructions in the link below
+https://www.digitalocean.com/community/tutorials/how-to-use-postgresql-with-your-django-application-on-ubuntu-14-04
+
```

Comment:
Hash is for Markdown headings. Try the attached file: [README.md.txt](https://github.com/killer-geckos/ivory-beam/files/4373938/README.md.txt)